packages/elements/src/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -305,7 +305,6 @@ Array [
   "textSecondary",
   "themes",
   "toggleOff",
-  "unstable_metadata",
```

Comment:
This doesn't look correct. I'd try updating snapshots again after pulling in latest from `main`